### PR TITLE
Feature/jo/fdp storage

### PR DESCRIPTION
**Pull request by jayolin**

### CODE REVIEW

File: src/components/Forms/ConfirmMnemonic/ConfirmMnemonic.tsx [hunk 1]
```diff
@@ -1,154 +1,144 @@
...
+  const [selected, setSelected] = useState<string[]>([]);
+
+  const words = useMemo<Array<string>>(
+    () => shuffleArray(mnemonic.split(' ')),
+    [mnemonic]
+  );
+
+  // const onSubmit = (data: ConfirmMnemonic) => {
+  //   const user = JSON.parse(localStorage.getItem('registerUser'));
+  //   const mnemonic = localStorage.getItem('registerMnemonic');
```
Comment (molekilla) [inline src/components/Forms/ConfirmMnemonic/ConfirmMnemonic.tsx:25]: remove commented out code

Comment (molekilla) [reply src/components/Forms/ConfirmMnemonic/ConfirmMnemonic.tsx:25]: or add a comment, I guess is the pending pod integration

File: src/components/Forms/LoginForm/LoginForm.tsx [hunk 2]
```diff
@@ -13,39 +13,60 @@ import { AuthenticationHeader } from '@components/Headers';
...
+      console.log(wallet);
+      router.push('/overview');
+    } catch (error) {
+      console.log(error);
+      setErrorMessage(error.message);
+    } finally {
+      setLoading(false);
+    }
+
+    // login(data)
```
Comment (molekilla) [inline src/components/Forms/LoginForm/LoginForm.tsx:48]: this can be removed

File: src/context/FdpStorageContext.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  process.env.NEXT_PUBLIC_RPC_URL as string
+);
+
+const fdpClient = new FdpStorage(
+  process.env.NEXT_PUBLIC_BEE_URL,
+  process.env.NEXT_PUBLIC_BEE_DEBUG_URL
+  // {
+  //   ensOptions: {
+  //     rpcUrl: process.env.NEXT_PUBLIC_RPC_URL,
+  //     contractAddresses: {
```
Comment (molekilla) [inline src/context/FdpStorageContext.tsx:17]: this can be cleaned up

File: src/pages/_app.tsx [hunk 4]
```diff
@@ -7,20 +7,55 @@ import { SearchProvider } from '@context/SearchContext';
...
+import { BeeDebug, DebugPostageBatch } from '@ethersphere/bee-js';
+import { useEffect } from 'react';
+
+async function sleep(ms = 1000): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+// async function testsSetup(): Promise<void> {
+//   if (!process.env.BEE_POSTAGE) {
+//     try {
```
Comment (molekilla) [inline src/pages/_app.tsx:21]: add a comment on how we use this postage batch helper

File: src/pages/register/payment/index.tsx [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+  const register = async () => {
+    try {
+      setStatusMessage('Registering user...');
+      // const wallet = Wallet.fromMnemonic(mnemonic);
+      fdpClient.account.setActiveAccount(wallet);
+      await fdpClient.account.register(username, password);
+
+      // create pods
+      // await Promise.all([
```
Comment (molekilla) [inline src/pages/register/payment/index.tsx:69]: this is also pending pods api working

File: src/pages/register/payment/index.tsx [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+      setTimeout(() => setCopyComplete(false), 4000);
+    } catch (error) {
+      console.log('Could not copy wallet address!');
+    }
+  };
+
+  const register = async () => {
+    try {
+      setStatusMessage('Registering user...');
+      // const wallet = Wallet.fromMnemonic(mnemonic);
```
Comment (molekilla) [inline src/pages/register/payment/index.tsx:64]: this can be removed

**Review by molekilla — CHANGES_REQUESTED** [review]

**Review by molekilla — APPROVED** [review]

**Comment by molekilla** [comment]
@crtahlin please approve and merge

**Comment by crtahlin** [comment]
I am not technical enough to review code; @tfius please review.

**Comment by molekilla** [comment]
@tfius let me know, need this code review to continue with the other issues

**Pull request merged**